straxen/plugins/peak_processing.py

```diff
@@ -146,12 +152,27 @@ class PeakPositions(strax.Plugin):
     # except for huge chunks
     parallel = False
 
-    __version__ = '0.1.0'
+    __version__ = '0.1.1'
 
     def setup(self):
         import tensorflow as tf
         keras = tf.keras
-
+        
+        # XENONnT
+        if self.config['nn_file']:
+            downloader = straxen.MongoDownloader()
+            tf.keras.backend.set_floatx('float32')
+            
+            nn_file = downloader.download_single(self.config['nn_file'])
+            
+            with tempfile.TemporaryDirectory() as tmpdirname:
```

Comment:
Do we also have to take out the bad PMTs like we did for 1T, I think so right?

Reply:
I'm not sure what's the best way to handle bad PMTs. We could just leave them as 0 in the input.

For 1T the bad PMTs are removed because the model were built that way.

Reply:
But this is something we would want to be able to specify at this stage as well right? It's easier than having to go back all the way to records.